adam-core/src/main/scala/org/bdgenomics/adam/rdd/BroadcastRegionJoin.scala

```diff
@@ -165,7 +165,7 @@ object BroadcastRegionJoin extends RegionJoin {
  *
  * @param regions The input-set of regions.
  */
-private[rdd] class NonoverlappingRegions(regions: Iterable[ReferenceRegion]) extends Serializable {
```

Comment:
Yeah, couldn't decide what to do with this.  `models` doesn't exactly seem like the right place for it IMO either.  What about `org.bdgenomics.adam.util`?